src/main.rs

```diff
@@ -0,0 +1,78 @@
+use k2::{
+    config::Config,
+    experiment::{Benchmark, ExperimentBuilder},
+    lang_impl::{self, Lang, LangImpl},
+    limit::Limit,
+    writer::{CsvWriter, KrunV1Writer, ResultWriter}
+};
+
+use clap::{App, Arg};
+
+use std::path::Path;
+
+fn main() {
+    let exp = setup().build();
+    let result = exp.run();
+    match result {
+        Ok(result) => {
+            CsvWriter::write(&Path::new("./result.csv"), &result);
+            KrunV1Writer::write(&Path::new("./result.krun"), &result);
+        }
+        Err(_error) => {
```

Comment:
I think for a simple example it's fine for this to be `unwrap` (or `expect` if you enjoy typing :) ).

Reply:
I actually prefer `expect`, as it gives you a string to grep for when debugging ;)

Reply:
`unwrap` reports a file name and a line number, so I don't think `expect` makes debugging any easier?

Reply:
> unwrap reports a file name and a line number

It does, but it's not a useful one:
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:374:21
```

*Sometimes* enabling the backtrace tells you the right line, but now always.

Reply:
Also, by using your own panic string, you can report the error using language which makes more sense to the function of the program (using domain knowledge). e.g.

```
Missing ELF section
```

v.s.

```
unwrap() on a None value.
```

I find the former more useful.